.mergify.yml

```diff
@@ -64,13 +70,15 @@ pull_request_rules:
 
   - name: comment when a pull request cannot be merged
     conditions:
+      - -draft
```

Comment:
cool, so there will be no duplicate comments, right?

Reply:
That means it won't approve any WIP pr.

Reply:
want to know  it will fix the duplicate comment,right, see  
![image](https://github.com/xline-kv/Xline/assets/28711504/71083a87-60ba-450f-b9e2-029e447b6970)

Reply:
But they are two different comments.

Reply:
I just send the wrong screen (:

it's 
![02b8f342f9ff48a1e63a6418ff66d03](https://github.com/xline-kv/Xline/assets/28711504/12c7850e-240b-403b-815f-703b7b02440c)


after talking with @Phoenix500526 , it will fix it.